README.md

```diff
@@ -21,8 +21,17 @@ import InfinityRoute from "ember-infinity/mixins/route";
 
 export default Ember.Route.extend(InfinityRoute, {
   model() {
+    var args = {
+      // Load 12 records in one group.
+      perPage: 12,
+      // Starting from page 1.
+      startingPage: 1,
+      /* Access model on 'controller.model'. This is required only if you place your model
+         on different location (when overriding setupController method for example). */
+      modelPath: 'controller.model'
```

Comment:
I like how you've lay'd this out, but can we plz move this to the advance usage section?